### PR TITLE
Add text to player Ui.

### PR DESCRIPTION
**Pull request by yonshavit**

### CODE REVIEW

**Comment by yonshavit** [comment]
I removed a file that I'm pretty sure should be ignored. please double check me on that.
Also - Is there any way to actually review changes to `SampleScene` ? Other than looking at the editor and hoping you actually see the differences?

File: Assets/Scripts/UIManagerScript.cs [hunk 1]
```diff
@@ -98,10 +105,14 @@ public class UIManagerScript : MonoBehaviour {
...
   public void UpdateUIOverlay() {
+    string barUiFormat = "{0}: {1:f2}";
     healthBar.fillAmount = Player.Instance.CurrentHealth / Player.Instance.FullHealth;
+    healthText.text = string.Format(barUiFormat,"Health", Player.Instance.CurrentHealth);
     shieldBar.fillAmount = Player.Instance.Shield.CurrentStrength / Player.Instance.Shield.MaxStrength;
+    shieldText.text = string.Format(barUiFormat, "Shield", Player.Instance.Shield.CurrentStrength);
     energyBar.fillAmount = Player.Instance.Reactor.CurrentEnergyLevel / Player.Instance.Reactor.MaxEnergyLevel;
-  }
+    energyText.text = string.Format(barUiFormat, "Energy", Player.Instance.Reactor.CurrentEnergyLevel);
+    }
```
Comment (nihohit) [inline Assets/Scripts/UIManagerScript.cs:115]: fix indentation.

**Review by nihohit — APPROVED** [review]
Fix indentation, or better yet, find why the IDE didn't fix it automatically.

**Comment by nihohit** [comment]
> I removed a file that I'm pretty sure should be ignored.

OK, then I'll add it to gitignore :)

> Is there any way to actually review changes to SampleScene ? Other than looking at the editor and hoping you actually see the differences?

With a lot of patience. In other words - not really :(

**Pull request merged**